src/uberComponents/ResourceChartingPage/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { Bundle, Resource } from 'fhir/r4b';
+import React from 'react';
+import { useParams, Route, Routes, useLocation, Link, useNavigate } from 'react-router-dom';
+
+import { RenderRemoteData, useService, WithId } from '@beda.software/fhir-react';
+
+import { RouteItem } from 'src/components/BaseLayout/Sidebar/SidebarTop';
+import { Tabs } from 'src/components/Tabs';
+import { getFHIRResources } from 'src/services';
+import { compileAsFirst } from 'src/utils';
+
+import { PageChartingContainer } from './PageChartingContainer';
+import { PageTabsProps, ResourceChartingPageProps } from './types';
+import { RecordType } from '../ResourceListPage/types';
+
+export function PageTabs<R extends Resource>({ tabs }: PageTabsProps<R>) {
```

Comment:
- [ ] Try to use PageTabs from ResourceDetailPage, refactor it if needed to make reusable